### ISSUE TITLE
`tabs_in_doc_comments` suggests me to use spaces for indentation

### ISSUE THREAD

**Issue by piegamesde** [rust-lang/rust-clippy#7309]
Lint name: `tabs_in_doc_comments`

I tried this code:

```rust
	/**
	 * Some code documentation
	 * string.
	 */
```

I expected to see this happen: The lint only activates when using tabs *within* the comment block, and not to indent the block.

Instead, this happened: I get a lint suggesting to replace my tabs with spaces. Doing this would result in mixed indentation (I generally use tabs and the lines with the comment would use spaces instead), which is bad.

### Meta

- `cargo clippy -V`: `clippy 0.1.51 (2fd73fa 2021-03-23)`
- `rustc -Vv`:
  ```
  rustc 1.51.0 (2fd73fabe 2021-03-23)
  binary: rustc
  commit-hash: 2fd73fabe469357a12c2c974c140f67e7cdd76d0
  commit-date: 2021-03-23
  host: x86_64-unknown-linux-gnu
  release: 1.51.0
  LLVM version: 11.0.1
  ```
